keps/sig-cli/2906-kustomize-function-catalog/README.md

```diff
@@ -743,30 +788,32 @@ definitions:
         type: array
         items:
           type: string
-  KRMFunction:
+  KRMFunctionDefinition:
```

Comment:
Why do we want to change the name here?

Reply:
As part of strengthening the analogy to CRD, I though it would be useful to include "Definition" in the name of this type. In the same way that the CRD kind defines a CR rather than being one itself, this type defines a function. Does that make sense?